### ISSUE TITLE
Toast designs

### ISSUE THREAD

**Issue by adamgall** [decentdao/decent-interface#1689]
**Toast designs**
Can we move away from the current Toast design?
[In Design system](https://www.figma.com/file/ur5hElxplkmlY9gnGKlwrm/Components-%26-Styles---Decent-Design-System-1.0.0?type=design&node-id=2179%3A26573&mode=design&t=WIRcqCxDLv8aVttV-1)
![image](https://github.com/decentdao/fractal-interface/assets/18615633/773394fb-260b-48d8-b04e-b75bdcd34895)
_Legacy design shown at bottom_

_Originally posted by @xraystyle1980 in https://github.com/decentdao/fractal-interface/issues/1578#issuecomment-2101329577_

Also, we've been asked by @nicolaus-sherrill to update the toast library to https://github.com/emilkowalski/sonner

**Comment by Da-Colon** [comment]
Yeah I asked for a easy redesign for the first pass of reskin. But these definitely can be focused in on.

**Comment by adamgall** [comment]
I think this needs a lot more thought before we can implement. We currently don't have any "levels" applied to our toasts. They're all the same. No difference between success and error.

In the spirit of keeping this reskin just to a reskin, and completing this delivery in time, I'm comfortable with redesigning our current catch-all toast. But I think it's out of scope to implement multiple designs and then go through the app applying deciding which flavor to give each toast. That should go into the backlog.

@xraystyle1980 @tomstuart123 thoughts?

**Comment by tomstuart123** [comment]
agreed. 

@xraystyle1980 lets assume we'll pick up this 'personalised toaster scheme' as an issue further down the line.

Does the current 'generic toaster' still need an update in your eyes ? Or is the current fine for now

**Comment by tomstuart123** [comment]
Still think this should stay deprioritised until more progress on hats / whitelisting proposals. Soon after though so I'll move it to a P1 but still in the backlog

**Comment by tomstuart123** [comment]
Design required

I would say now we have hats progress, this COULD (but lets see about SHOULD) be an option to be picked up before design moves onton whitelisting
